atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraKeyValueServices.java

```diff
@@ -59,7 +59,19 @@ private CassandraKeyValueServices() {
         // Utility class
     }
 
-    static void waitForSchemaVersions(Cassandra.Client client, String tableName, int schemaTimeoutMillis)
+    static void waitForSchemaVersions(
+            Cassandra.Client client,
+            String tableName,
+            int schemaTimeoutMillis)
+            throws TException {
+        waitForSchemaVersions(client, tableName, schemaTimeoutMillis, false);
```

Comment:
all calls to this besides the failFastInInit are real schema changes.

Technically we could allow things like createTable("foo") to succeed with a dead node, however if that node comes up in the future and someone issues a read or write against that table before [node who just came up] updates his schema and creates the table locally, he'll throw an exception.

I'm okay with doing that, it'll just be a pain instrumenting all of the places we could possibly touch tables with a proxy or something that provides exception handling for this.